modules/phase_field/src/materials/ComputePolycrystalElasticityTensor.C

```diff
@@ -48,44 +48,45 @@ ComputePolycrystalElasticityTensor::computeQpElasticityTensor()
   const std::vector<std::pair<unsigned int, unsigned int> > & active_ops = _grain_tracker.getElementalValues(_current_elem->id());
   unsigned int n_active_ops = active_ops.size();
 
-  if (n_active_ops < 1 && _t_step > 0)
-    mooseError("No active order parameters");
-
   // Calculate elasticity tensor
   _elasticity_tensor[_qp].zero();
-  Real sum_h = 0.0;
-  for (unsigned int op = 0; op < n_active_ops; ++op)
-  {
-    const unsigned int grain_index = active_ops[op].first;
-    const unsigned int op_index = active_ops[op].second;
-
-    // Interpolation factor for elasticity tensors
-    Real h = (1.0 + std::sin(libMesh::pi * ((*_vals[op_index])[_qp] - 0.5))) / 2.0;
-
-    // Sum all rotated elasticity tensors
-    _elasticity_tensor[_qp] += _grain_tracker.getData(grain_index) * h;
-    sum_h += h;
-  }
-
-  const Real tol = 1.0e-10;
-  sum_h = std::max(sum_h, tol);
-  _elasticity_tensor[_qp] /= sum_h;
 
   // Calculate elasticity tensor derivative: Cderiv = dhdopi/sum_h * (Cop - _Cijkl)
   for (unsigned int op = 0; op < _op_num; ++op)
     (*_D_elastic_tensor[op])[_qp].zero();
 
-  for (unsigned int op = 0; op < n_active_ops; ++op)
+  if (n_active_ops > 0)
```

Comment:
This optimization does not seem necessary and will actually penalize the the regular case. I suggest just removing the error.